evennia/commands/default/building.py

```diff
@@ -2350,6 +2352,8 @@ def func(self):
                 else:
                     string = "|wOne Match|n(#%i-#%i%s):" % (low, high, restrictions)
                     string += "\n   |g%s - %s|n" % (results[0].get_display_name(caller), results[0].path)
+                    if "locate" in self.cmdstring and nresults == 1 and results[0].location:
+                        string += "  Location: {}".format(results[0].location.get_display_name(caller))
```

Comment:
Looking at this, having just a few extra blank spaces looks a little inconsistent with other code. I think it's better to put this in a parenthesis like so: `" (location: {}"`. Also, we use `|w` color for the instruction and green for the match, so we should use that here for the location too.

Reply:
`string += "  (|wlocation|n: |g{})|n".format(results[0].location.get_display_name(caller))`

Reply:
Yeah, looks good. :)